tests/integration/test_enrollment/data/wazuh_enrollment_tests.yaml

```diff
@@ -0,0 +1,273 @@
+-
```

Comment:
We need to change the quotes to follow the next format:

```
x = f"{var}string"
x = 'raw string'
x = "this is my raw 'string' with single quotation marks"
```

Reply:
As this is a yaml file I think it doesn´t follow this convention.